Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -148,7 +148,8 @@ def http_request(self, method, url_suffix=None, full_url=None, params=None, data
         """
         if not full_url:
             params = params or {}
-            params['api-version'] = API_VERSION
+            if not params.get('api-version'):
```

Comment:
Why do we need this condition? do we expect API version param?

Reply:
same as in key vault, this is the http_request function params, not the demisto.params()

Reply:
I can see that the API version of the new lost commands is passed as part of the full URL. In that case why do we need this condition? 
`if not params.get('api-version'):`

Reply:
Fixed.